src/components/article/ListPagination.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Pagination, Grid } from 'semantic-ui-react';
+import '../../styles/Articles.scss';
+
+
+class ListPagination extends Component {
+    static currentLocalStorage = (localStorage) => {
+        let currentPage = localStorage.getItem('page');
+        if (currentPage !== null) {
+            return currentPage;
+        }
+    }
+    onPageChange = (event, data) => {
+        const { fetchArticles } = this.props;
+        const url = 'https://ah-backend-summer-staging.herokuapp.com/api/v1/articles';
```

Comment:
I think we should place the base URL(`https://ah-backend-summer-staging.herokuapp.com`) constant to a common place to avoid duplication.

Reply:
Yep, I agree, we will tackle it when restructuring since its used in a few places...maybe in an env file.